### PR TITLE
add to docs how to use sequence to fetch doc

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

File: docs/catalog.md [hunk 1]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+```
+function main(doc){
+  return { "isWalter:" : doc.name === "Walter White"};
+}
+```
+```
+$ wsk action create myAction myAction.js
+```
+
+Create an action sequence that get's the document first, then calls your current `myAction` that expects a document as input
```
Comment (rabbah) [inline docs/catalog.md]: get's -> gets

File: docs/catalog.md [hunk 2]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
   }
   ```
 
+### Using an action sequence and a change trigger to get a document from a Cloudant database
```
Comment (rabbah) [inline docs/catalog.md]: get -> process?
it's not clear why i'd need a sequence just to get the document - it's really only necessary to operate on the document.

File: docs/catalog.md [hunk 3]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+
+You can use an action sequence in a rule to fetch the document associated with a Cloudant change event.
```
Comment (rabbah) [inline docs/catalog.md]: to fetch and process the document ...

File: docs/catalog.md [hunk 4]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+
+Create an action that process a document from Cloudant, it will expect a document as a parameter.
```
Comment (rabbah) [inline docs/catalog.md]: process -> will process.

File: docs/catalog.md (hunk 1 above)
Comment (rabbah) [inline docs/catalog.md]: perhaps prefix by: To read the document from the database, you can use the `read` action in the cloudant package. 

then lead into the sequencing.

File: docs/catalog.md [hunk 5]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+
+Now create a rule that associates your trigger with the new action `sequenceAction`
+```
+$ wsk rule create myRule myCloudantTrigger sequenceAction
+```
+
+The action sequence will need to know the database name to fetch the document from.
+Set a parameter on the trigger for `dbname`
+```
+$ wsk trigger update myCloudantTrigger --param dbname testdb
```
Comment (rabbah) [inline docs/catalog.md]: This is the temporary part right? (the dbname will come with the feed once you make the changes there?).

File: docs/catalog.md [hunk 6]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+```
+
+**Note** The Cloudant change trigger used to support the parameter `includeDocs`, this is not longer supported.
+  To migrate an old trigger that was created with `includeDocs`:
+  Recreate the trigger with no includeDocs parameter
```
Comment (rabbah) [inline docs/catalog.md]: To migrate an old trigger that was created with `includeDocs`:
+  Recreate the trigger with no includeDocs parameter

-> 

You will need to recreate triggers previously created with `includeDocs`:

File: docs/catalog.md (hunk 2 above)
Comment (csantanapr) [inline docs/catalog.md]: I had problems coming with a good title. yeah I think s/get/process will do the trick

File: docs/catalog.md (hunk 5 above)
Comment (csantanapr) [inline docs/catalog.md]: yes temporary

**Comment by csantanapr** [comment]
@rabbah take a look at 0f2167f62e5744b3a3cb3a530182833f0b392427 I tried to address your review comments.

File: docs/catalog.md [hunk 7]
```diff
@@ -191,6 +191,46 @@ You can use an action to fetch a document from a Cloudant database called `testd
...
+```
+
+The action sequence will need to know the database name to fetch the document from.
+Set a parameter on the trigger for `dbname`
+```
+$ wsk trigger update myCloudantTrigger --param dbname testdb
+```
+
+**Note** The Cloudant change trigger used to support the parameter `includeDocs`, this is not longer supported.
+  You will need to recreate triggers previously created with `includeDocs`:
```
Comment (jasonpet) [inline docs/catalog.md]: The parameter name in the cloudant changes feed was actually called includeDoc.

Comment (csantanapr) [inline docs/catalog.md]: good catch

**Comment by csantanapr** [comment]
@jasonpet your comments addressed here 1d73a610cd2eddb077e48e17e5705a5ff163f536

**Comment by rabbah** [comment]
Removed update trigger bits (not needed anymore per @jasonpet and @csantanapr ), and merged as https://github.com/openwhisk/openwhisk/commit/548dadd878eaaf71155664a1434883db2e72a923.

**Pull request closed without merging**